matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/SessionModule.kt

```diff
@@ -74,6 +81,7 @@ internal abstract class SessionModule {
         @JvmStatic
         @UserId
         @Provides
+        @SessionScope
```

Comment:
Why this is not applicable to all fun annotated with `@Provides` in this class?

Reply:
Only the one with computations or logic (maybe I've already added on too many)